src/github/orbit-helpers.js

```diff
@@ -323,3 +321,38 @@ export function _getRepositoryFullName() {
     window.location.pathname.split("/")[2]
   }`;
 }
+
+/**
+ * Fetches chunked repositories from chrome storage
+ *
+ * @returns Array<String> a 1d array of all repsoitory names, ie ["repo-1", "repo-2"]
+ */
+export async function _fetchRepositories() {
+  const { repository_keys } = await chrome.storage.sync.get("repository_keys");
+
+  // Backwards compatibility - if we do not have repository keys,
+  //  default to how we used to store them
+  if (repository_keys === undefined) {
+    const { repositories } = await chrome.storage.sync.get({
+      repositories: "",
```

Comment:
Should the default value be an empty array, rather than an empty string? It looks like the other code path returns an array